addons/calendar/models/calendar.py

```diff
@@ -997,16 +1013,24 @@ def get_recurrent_ids(self, domain, order=None):
                 ok = True
                 for arg in domain:
                     if str(arg[0]) in ('start', 'stop', 'final_date'):
+                        if str(arg[0]) == 'start':
+                            r_date = r_start_date
+                        else:
+                            r_date = r_stop_date
+                        if len(arg[2]) > len(r_date.strftime(DEFAULT_SERVER_DATE_FORMAT)):
```

Comment:
In which case do you get this? We are talking about recurrent events so you have a start and stop date.